docs/model-archiver.md

```diff
@@ -117,7 +117,7 @@ optional arguments:
 
 A valid model name must begin with a letter of the alphabet and can only contains letters, digits, underscores `_`, dashes `-` and periods `.`.
 
-**Note**: The model name can be overridden when you register the model with [Register Model API](../docs/management_api.md#register-a-model).
+**Note**: The model name can be overridden when you register the model with [Register Model API](https://pytorch.org/serve/management_api.html#register-a-models).
```

Comment:
Were you able to get this to work when running locally?
When I tried, it incorrectly resolves to `https://pytorch.org/serve/management_api.md#register-a-models` (has a .md instead of .html). I think it's because sphinx for .md doesn't support linking to a subheader in another file.

Reply:
I wasn't able to get that new syntax `[Register Model API](management_api.md#register-a-models)` to resolve correctly - this new syntax resolves to resolves `https://pytorch.org/serve/management_api.mdl#register-a-models` instead of a .html file.
Were you able to get this to work when you built it?
What I'd used in the index.rst was the [sphinx autosectionlabel directive](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html) but I think it only applies to .rst files.